### PR TITLE
Feat/import csv existing

### PR DESCRIPTION
**Pull request by joshenlim**
### Users can import data via CSV into the table via the dropdown here
image

### Users can choose which columns to import from the CSV
There will be an indication if the imported CSV is incompatible with the table that it's going to be imported to, or if there are any issues with the imported CSV
These will be the same for when users are importing a CSV while creating a new table
image

### CODE REVIEW

File: studio/components/interfaces/TableGridEditor/SidePanelEditor/SpreadsheetImport/SpreadsheetImportPreview.tsx [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+            }
+            className="px-1"
+            onClick={() => setExpandPreview(!expandPreview)}
+          />
+        </div>
+      </div>
+      <div
+        style={{ maxHeight: expandPreview ? '1000px' : '0px' }}
+        className="transition-all overflow-y-hidden"
+      >
```
Comment (MildTomato) [inline studio/components/interfaces/TableGridEditor/SidePanelEditor/SpreadsheetImport/SpreadsheetImportPreview.tsx:84]: there is a component for this @joshenlim 

`<Collapsible>`

**Review by MildTomato — CHANGES_REQUESTED** [review]

File: studio/components/interfaces/TableGridEditor/SidePanelEditor/SpreadsheetImport/SpreadsheetImportPreview.tsx (hunk 1 above)
Comment (MildTomato) [reply studio/components/interfaces/TableGridEditor/SidePanelEditor/SpreadsheetImport/SpreadsheetImportPreview.tsx:84]: example
https://github.com/supabase/supabase/blob/master/packages/ui/src/components/Collapsible/Collapsible.stories.tsx

Comment (joshenlim) [reply studio/components/interfaces/TableGridEditor/SidePanelEditor/SpreadsheetImport/SpreadsheetImportPreview.tsx]: just pushed an update!

File: studio/components/interfaces/TableGridEditor/SidePanelEditor/SidePanelEditor.tsx [hunk 2]
```diff
@@ -394,6 +392,54 @@ const SidePanelEditor = ({
...
+      ui.setNotification({
+        id: toastId,
+        category: 'success',
+        message: `Successfully imported ${rowCount} rows of data into ${selectedTable.name}`,
+      })
+      await Promise.all([
+        queryClient.invalidateQueries(
+          sqlKeys.query(project?.ref, [selectedTable!.schema, selectedTable!.name])
+        ),
+      ])
```
Comment (alaister) [inline studio/components/interfaces/TableGridEditor/SidePanelEditor/SidePanelEditor.tsx:437]: Worth swapping the order of these two? So the toast shows after everything finishes loading

**Review by alaister — APPROVED** [review]
LGTM! Would you also mind merging in master to make sure the new `package-lock.json` changes still build correctly?

File: apps/www/components/LaunchWeek/LaunchSection/TicketBrickWallSlider.tsx [hunk 3]
```diff
@@ -29,7 +29,7 @@ export function TicketBrickWallSlider({ users, reverse, speed = 10000 }: Props)
 
   useEffect(() => {
     // trigger autplay if viewport resize
```
Comment (alaister) [inline apps/www/components/LaunchWeek/LaunchSection/TicketBrickWallSlider.tsx:31]: `autplay` -> `autoplay` in the comment too!

**Review by alaister — APPROVED** [review]

**Review by MildTomato — APPROVED** [review]

**Pull request merged**